src/pizzeria/TestOrders.java

```diff
@@ -4,11 +4,12 @@
     public static void main(String[] args) {
         CheckPrinter checkPrinter = new CheckPrinter();
         Customer customer = new Customer(7717, "Heghine","+(374 94)374374");
-        Ingredient ingredientsToAdd = new Ingredient("Pepper", 0.4);
-        Ingredient[] ingredients = {ingredientsToAdd, Ingredient.getCheese(), Ingredient.getCorn(), Ingredient.getTomatoPaste()};
-        PizzaType pizzaType = new PizzaType("Calzone");
+        Ingredient[] ingredients = {Ingredient.CHEESE, Ingredient.CORN, Ingredient.TOMATO_PASTE};
+        PizzaType pizzaType = PizzaType.CLOSED;
```

Comment:
No need to create a variable here. You can directly use enum inside **new Pizza();**

Reply:
done